Package.swift

```diff
@@ -35,10 +35,13 @@ let package = Package(
                     package: "swift-transformers",
                     moduleAliases: ["Models": "TransformersModels"]
                 ),
-            ],
+            ]/*,
+              * Uncomment this line to load models for tests
+              *
             resources: [
-                .copy("Resources"),
+                .copy("Resources/gemma-2-2b-it-4bit"),
```

Comment:
```suggestion
                .copy("Resources"),
```

Reply:
We can't do that. The library won't codesign when building for iOS because the way `.copy()` works for macOS bundles and iOS bundles is different. If we don't specify the subdirectory names on iOS, the bundle format isn't valid on iOS.